dynamodb/src/main/scala/zio/dynamodb/DynamoDBExecutorImpl.scala

```diff
@@ -261,19 +243,20 @@ private[dynamodb] final case class DynamoDBExecutorImpl private (dynamoDb: Dynam
       .map(chunk => (chunk, chunk.lastOption))
 
   private def generateUpdateItemRequest(updateItem: UpdateItem): UpdateItemRequest = {
-    val maybeUpdateConditionExpr =
-      zipOptionalRenderables(Some(updateItem.updateExpression), updateItem.conditionExpression)
+    val maybeUpdateAndConditionExpr: (AliasMap, (String, Option[String])) = (for {
```

Comment:
```suggestion
    val (aliasMap, (updateExpression, conditionExpression)) = (for {
```
could something like this be used?